analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -383,13 +384,13 @@ open class Npm(
         val nodeModulesDir = moduleDir.resolve("node_modules")
         if (!nodeModulesDir.isDirectory) return emptyList()
 
-        val searchDirs = nodeModulesDir.listFiles().filter { file ->
-            file.isDirectory && file.name.startsWith("@")
-        } + nodeModulesDir
+        val searchDirs = nodeModulesDir.listFiles(FileFilter {
```

Comment:
commit-msg: generating **is** -> **it** ?

Reply:
Done.